packages/terra-form-select/src/_Frame.jsx

```diff
@@ -429,6 +445,7 @@ class Frame extends React.Component {
                  onDeselect,
                  optionFilter,
                  noResultContent,
+                 menuTriggerElement: variant === Variants.DEFAULT ? this.select : this.input,
```

Comment:
This name feels a little awkward to me. I don't have a great alternative but maybe something along the lines of `focusTarget`, `focusRegion`, `focusTrap`, etc... or something that implies the inverse of aria-owns / aria-controls. `parent`